x-pack/plugins/security_solution/public/management/cypress.config.ts

```diff
@@ -33,24 +35,27 @@ export default defineCypressConfig({
   experimentalStudio: true,
 
   env: {
-    KIBANA_URL: 'http://localhost:5601',
-    ELASTICSEARCH_URL: 'http://localhost:9200',
-    FLEET_SERVER_URL: 'https://localhost:8220',
-    // Username/password used for both elastic and kibana
+    'cypress-react-selector': {
```

Comment:
I think we can remove it as we are not using `cypress-react-selector`, I believe